css/index.css

```diff
@@ -77,13 +80,14 @@ header {
 }
 
 header nav {
-    margin-top: 40px; 
-    width: 610px;
+    margin-top: 5%;
```

Comment:
I believe I mentioned this in our channel's meeting yesterday but margins regarding top/bottom are most likely going to be in px rather than percentages (unless stated otherwise). Top/bottom are more of aesthetic views rather than fluid design. But on saying that, converting px to % can be tedious, I'm glad to see that you have taken the time to look over and adjust accordingly!